src/main/webapp/app/js/redirects.js

```diff
@@ -52,4 +52,12 @@ function switchPersonalData() {
     } else {
         block.classList.add("w3-hide");
     }
+}
+
+function displayError(response, expected) {
+    if (response.status === expected) {
+    } else {
+        alert("Something went wrong! error: " + response.status.toString());
+        history.go(-1);
```

Comment:
What is the reason for automatically going back to the previous page in this case?